frontend/src/scenes/session-recordings/playlist/SessionRecordingsPlaylistItem.tsx

```diff
@@ -39,7 +39,7 @@ export function SessionRecordingPlaylistItem({
             : recording.person?.properties || {}
 
     const propertyIcons = (
-        <div className="flex flex-row flex-nowrap shrink-0 gap-1 h-6">
+        <div className="flex flex-row flex-nowrap shrink-0 gap-1 h-6 ph-no-capture">
```

Comment:
why no capture? Doesn't feel like this is particularly sensitive?

Reply:
My understanding was that we don't necessarily want to capture the PII of our user's users?